Update stats and download version

huey735 opened the pull request:
@ripcurlx there are now 4 additional download pages on the website:
* /de/downloads.html
* /es/downloads.html
* /pt-PT/downloads.html
* /zh-CN/downloads.html

@m52go going forward, please also update the /_data/strings_en.yml when doing changes like updating the stats. The strings inside /_data/strings_en.yml are distributed into the following lists:
* footer
* homepage_content
* main_nav
* os_selector_options:
* dao
* downloads
* faq
* markets
* stats

I'll be setting up Transifex so that the "strings" resource updates when there are changes to  /_data/strings_en.yml.

huey735 committed:
Update stats and download version

huey735 committed:
Add outdated translation alert tool

m52go commented on _includes/main_nav_tr.html:
```diff
@@ -1,7 +1,14 @@
+      <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
 
+      <!-- Outdated translation tag-->
+      {% if page.lang == page.lang == "de" or page.lang="es" or "pt-PT" or page.lang == "zh-CN" %}
```
When is this "outdated translation" text supposed to show? This line seems to indicates it will show any time the current language is not EN (and indeed that's what happens when I run this locally).

Otherwise, please double-check the logic. `page.lang == page.lang` is redundant and you're missing the equality operator for ES (`page.lang="es"` needs `==`).

huey735 replied:
My idea is that we'd regularly update it by taking out and leaving the relevant ones.

m52go reviewed (changes requested):
Agreed about the stats numbers—will remember this next time.

That `version` front-matter variable is not used...the `site.client_version` is the way this page gets the version number. I'll make a PR to remove it altogether (done; see #206). Then we won't have to worry about updating those files at all :)

huey735 committed:
Correct logic of outdated translation alert tool

I corrected the logic of liquid flag for the outdated translation
alert tool and removed the 'version' variable from the front
matter of the download files in order to in agreement with
https://github.com/bisq-network/bisq-website/pull/206
